### PR TITLE
Adding New 3DS device and match to Netfront NX rules, plus test

### LINKED ISSUE #1236: Incorrect User Agent Parsing (New Nintendo 3DS)
Using https://browscap.org/ua-lookup, the following user agent is not correctly parsed.

`Mozilla/5.0 (New Nintendo 3DS like iPhone) AppleWebKit/536.30 (KHTML, like Gecko) NX/3.0.0.5.20 Mobile NintendoBrowser/1.8.10156.US`

It is parsed as an iPhone device when the parsing should be similar to 
`Mozilla/5.0 (Nintendo 3DS; U; ; fr) Version/1.7630.US`

Not sure if the device should be "3DS" or "New 3DS".

### PR DESCRIPTION
**Pull request by jaydiablo**
This fixes #1236

### CODE REVIEW

File: resources/user-agents/browsers/access-netfront/access-netfront-nx-generic.json [hunk 1]
```diff
@@ -28,6 +28,11 @@
           "device": "Nintendo 3DS"
         },
         {
+          "match": "Mozilla\/* (New Nintendo 3DS *) NX\/*",
```
Comment (mimmi20) [inline resources/user-agents/browsers/access-netfront/access-netfront-nx-generic.json]: Please expand this pattern with `AppleWebKit\/* (KHTML* like Gecko)` and add this pattern to the NX 3.0 too.

**Comment by jaydiablo** [comment]
Closing in favor of #1242 (sorry for the noise, trying to get rid of those merge commits).

@mimmi20 I'll address your changes in that PR, thanks.

**Comment by mimmi20** [comment]
Did you try to rebase the branch?

**Comment by jaydiablo** [comment]
I branched from my local master that had a merge commit (but I don't think I synched the merge commit to github).

I've since rebased my master branch, then branched the new branches from that, which seems to have done the trick.

**Pull request closed without merging**